python/cugraph/tests/dask/opg_pagerank_testing.py

```diff
@@ -43,7 +43,7 @@ def test_dask_pagerank():
     g.from_cudf_edgelist(df, 'src', 'dst')
 
     dg = cugraph.DiGraph()
-    dg.from_dask_cudf_edgelist(ddf)
+    dg.from_cudf_edgelist(ddf)
```

Comment:
Are we testing the multi-gpu path here since it is bringing the dask dataframe into a single gpu. We can instead perform g.from_cudf_edgelist(ddf, 'src', 'dst') at line 43

Reply:
good catch, I forgot to revert it after testing